src/main/java/seedu/address/model/record/RecordList.java

```diff
@@ -93,7 +96,15 @@ public boolean equals(Object other) {
                 && recordList.equals(((RecordList) other).recordList)); // state check
     }
 
+    /**
+     * Adds a Record to the list.
+     * The record must not already exist in the list.
+     */
```

Comment:
This javadocs might be wrong, should be delete record right?

Reply:
Yup, good catch!